src/Features/LanguageServer/ProtocolUnitTests/References/FindAllReferencesHandlerTests.cs

```diff
@@ -45,7 +49,44 @@ void M2()
             Assert.Equal("M", results[1].ContainingMember);
             Assert.Equal("M2", results[3].ContainingMember);
 
-            AssertValidDefinitionProperties(results, 0);
+            AssertValidDefinitionProperties(results, 0, Glyph.FieldPublic);
+        }
+
+        [WpfFact(Skip = "https://github.com/dotnet/roslyn/issues/43063")]
```

Comment:
does this need to be skipped still?

Reply:
I can't see the build results in the linked issue to see why they failed, but with the handler only supporting streaming I can't see how they ever passed, so was planning on unskipping when adding non-streaming support.